tests/src/test/java/alluxio/master/file/ConcurrentFileSystemMasterCreateTest.java

```diff
@@ -87,8 +86,10 @@ public static void afterClass() throws Exception {
   }
 
   @Before
-  public void before() {
+  public void before() throws Exception {
     mFileSystem = FileSystem.Factory.get();
+    // Mount the sleeping UFS, to always the sleeping UFS over the ufs cluster.
```

Comment:
Sure. I updated the comment to specify that these tests will only use the sleeping ufs.